### ISSUE TITLE
Feature: Create default KRI visualization `Visualize_Score()`

### ISSUE THREAD

**Issue by mattroumaya** [Gilead-BioStats/gsm#579]
## Feature Details
TLDR: Barchart with current score with thresholds overlaid, with one bar per site
- Will have a lot of bars, so it'll be necessary to have interactive/hover option
- Visualizing KRI instead of Score (so for `Analyze_Identity()`, we're always visualizing KRI)
- Includes upper and lower flagging threshold
  - Bar chart? Showing both the score (with thresholds), and another barchart with raw KRI (without thresholds). 
    - Toggle button for thresholds
  - Scatter plot with a diagonal line to indicate higher/lower threshold?
  - Scatter plot with Score on one axis and KRI on the other?
- Always run `Visualize_Score()` for all assessments, plus the option to run specific visualizations (`Visualize_Scatter()`/`Visualize_Count()`) for any given assessment.
- Do we always want to have flags for KRI? Or will we only have flags for the KRI when using `Analyze_Identity()`?
  - Most of the time, we won't have nominal thresholds. (special use case for QTLs, small sample size).
  - Can still highlight flagged sites with footnote to indicate flagging limitations.

## Possible Implementation
- For use in nominal assessments/when using `Analyze_Identity()`
